be/src/column/column_helper.cpp

```diff
@@ -211,32 +210,29 @@ ColumnPtr ColumnHelper::create_column(const TypeDescriptor& type_desc, bool null
     if (VLOG_ROW_IS_ON) {
         VLOG_ROW << "PrimitiveType " << type << " nullable " << nullable << " is_const " << is_const;
     }
-    if (nullable && is_const) {
-        return ColumnHelper::create_const_null_column(size);
-    }
-
-    if (type == TYPE_NULL) {
+    if (nullable || type == TYPE_NULL) {
         if (is_const) {
             return ColumnHelper::create_const_null_column(size);
-        } else if (nullable) {
-            return NullableColumn::create(BooleanColumn::create(), NullColumn::create());
+        } else {
+            return NullableColumn::create(BooleanColumn::create(size), NullColumn::create(size, DATUM_NULL));
```

Comment:
https://github.com/StarRocks/starrocks/blob/338fbe6c0832878aaf6a90520ca5961af737c6b9/be/src/serde/protobuf_serde.cpp#L129 

may be has problem here